application: serial_lte_modem: Update #XSLMVER command (#10324)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

* Updated:

  * The response for the ``#XDFUGET`` command, using unsolicited notification to report download progress.

>   * Report the version of |NCS| as well as the modem library in ``#XSLMVER``.

Reviewer: ```suggestion
  * The ``#XSLMVER`` command to report the versions of both the |NCS| and the modem library.
```
